docs/playbook.md

```diff
@@ -36,6 +36,14 @@ Command to enable the flag:
 
 Refer to know more about CGO: https://www.golinuxcloud.com/cgo-tutorial/
 
+### Mount gets stuck with error: DefaultTokenSource: google: could not find default credentials.
+This state comes when we don't provide key-file as a mounting argument. In this
+case GCSFuse tries to fetch the default application credential, but got stuck
+due to absense of application-credentials on the VM.
+
+Resolve this issue by fetching the creds with the below command:  
```

Comment:
Run this command to fetch default creds to the vm. This creds are copied to x place